src/app/[locale]/(logged-in)/organisation/profile/user-administration/[subTabId]/[id]/[subSubTabId]/components/OrganisationUser/OrganisationUser.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import {
+  PageColumnDetails,
+  PageBodyContainer,
+  PageColumnBody,
+  PageColumns,
+} from "@/modules";
+import { UserGroup } from "@/consts/user";
+import { useStore } from "@/data/store";
+import { useQuery } from "@tanstack/react-query";
+// import { getCustodianProjectUserValidationLogsQuery } from "@/services/validation_logs";
+import { getUserQuery } from "@/services/users";
+// import ActionValidationPanel from "@/modules/ActionValidationPanel";
+import { useTranslations } from "next-intl";
+import { notFound } from "next/navigation";
+import { useEffect } from "react";
+import { UserSubTabs } from "../../../../../../consts/tabs";
+import SubTabsSections from "../SubTabSections";
+import SubTabsContents from "../SubsTabContents";
+
+interface OrganisationUserProps {
+  userId: number;
+  subSubTabId: UserSubTabs;
+}
+
+const NAMESPACE_TRANSLATION_ORGANISATION_PROFILE = "ProfileOrganisation";
+
+function OrganisationUser({ userId, subSubTabId }: OrganisationUserProps) {
+  const t = useTranslations(NAMESPACE_TRANSLATION_ORGANISATION_PROFILE);
+
+  const { data: userData, isFetched } = useQuery(getUserQuery(+userId));
+
+  if (userData?.data.user_group !== UserGroup.USERS && isFetched) {
+    notFound();
+  }
+
+  // Commented out for now as the correct query is not available yet
```

Comment:
These bits are left in but commented out so they're there for @calmacx ' modifications in a later PR.